Rakefile

```diff
@@ -151,6 +151,7 @@ task :check_consistency_between_aws_and_carrenza do
     govuk::apps::ckan::redis_port
     govuk::apps::ckan::s3_aws_access_key_id
     govuk::apps::ckan::s3_aws_secret_access_key
+    govuk::apps::collections::feature_flag_accounts
```

Comment:
Does / should this file still do anything? Aren't we now fully off carrenza?

Reply:
Some Carrenza still exists (eg, deploy.publishing.service.gov.uk), hopefully it'll go away soon and huge swathes of puppet can be simplified.